vunit/ui.py

```diff
@@ -393,6 +394,41 @@ def add_external_library(self, library_name, path, vhdl_standard=None):
         self._project.add_library(library_name, abspath(path), vhdl_standard, is_external=True)
         return self.library(library_name)
 
+    def add_source_files_from_csv(self, project_csv_path, vhdl_standard=None):
+        """
+        Add a project configuration, mapping all the libraries and files
+        :param project_csv_path: path to csv project specification, each line contains the name 
```

Comment:
The documentation should also say that relative file names are relative to the parent folder of the csv file.